pages/admin/permit-holders.tsx

```diff
@@ -294,13 +343,23 @@ export default function PermitHolders() {
                   textAlign="left"
                   width="320px"
                 >
-                  <MenuText name={`Permit Status`} value={permitStatusFilter} />
+                  <MenuText
+                    name={`Permit Status`}
+                    value={permitStatusFilter && permitStatusText[permitStatusFilter]}
+                  />
                 </MenuButton>
                 <MenuList>
-                  <DropDownItems
-                    items={permitStatusOptions}
-                    setStateCallback={setPermitStatusFilter}
-                  />
+                  <MenuItem onClick={() => setPermitStatusFilter(undefined)}>All</MenuItem>
+                  {permitStatusOptions.map((value, i) => (
+                    <MenuItem
+                      key={`dropDownItem-${i}`}
+                      onClick={() => {
+                        setPermitStatusFilter(value);
```

Comment:
I think that we need to also reset the page number to 0 anytime a filter is applied

Reply:
yeah i missed that good catch

Reply:
When trying to implement this in the way you did it on the request table page I'm getting some weird results so I'm pairing with oustan later to figure it out.
It'll be added before merging